core/src/test/scala/org/elasticmq/actor/QueueActorMsgOpsTest.scala

```diff
@@ -147,7 +147,7 @@ class QueueActorMsgOpsTest extends ActorTest with QueueManagerForEachTest with D
     val q1 = createQueueData("q1", MillisVisibilityTimeout(1L))
 
     for {
-      Right(queueActor) <- queueManagerActor ? CreateQueue(q1)
+      Right(queueActor) <- queueManagerActor ? CreateQueue(q1.copy(maxReceiveCount = 2))
```

Comment:
yes, bc now number if receives is count and msg will not be available if we reach limit.